GUI/Main.cs

```diff
@@ -749,20 +749,19 @@ private void ModList_CellMouseDoubleClick(object sender, DataGridViewCellMouseEv
                 var module_installer = ModuleInstaller.GetInstance(CurrentInstance, GUI.user);
                 full_change_set = await mainModList.ComputeChangeSetFromModList(registry, user_change_set, module_installer, CurrentInstance.VersionCriteria());
             }
-            catch (InconsistentKraken)
-            {
-                // Need to be recomputed due to ComputeChangeSetFromModList possibly changing it with too many provides handling.
-                user_change_set = mainModList.ComputeUserChangeSet();
-                new_conflicts = MainModList.ComputeConflictsFromModList(registry, user_change_set, CurrentInstance.VersionCriteria());
-                full_change_set = null;
-            }
             catch (TooManyModsProvideKraken)
             {
                 // Can be thrown by ComputeChangeSetFromModList if the user cancels out of it.
                 // We can just rerun it as the ModInfoTabControl has been removed.
                 too_many_provides_thrown = true;
             }
-
+            // ComputeChangeSetFromModList returns null if an inconsistency was found, we need to highlight the inconsistencies.
+            if (full_change_set == null)
+            {
+                // Need to be recomputed due to ComputeChangeSetFromModList possibly changing it with too many provides handling.
+                user_change_set = mainModList.ComputeUserChangeSet();
+                new_conflicts = MainModList.ComputeConflictsFromModList(registry, user_change_set, CurrentInstance.VersionCriteria());
```

Comment:
`ComputeConflictsFromModList`'s type signature and approach are unsafe; it has a `ToDictionary` call on a list with potentially non-unique keys, which can throw an `ArgumentException`. Basically it can't represent conflicts between a given mod and multiple other mods, which this PR is trying to do (more). This would be another good spot to use `List<KeyValuePair<CkanModule, CkanModule>>`.

Reply:
So, I didn't actually change that code at all, and I'm not entirely sure how this task works or what  calls it. This is one of the main reasons I'm unsure about this change.